client/src/components/Logo/index.js

```diff
@@ -1,6 +1,8 @@
 import styled from "styled-components";
 
 const Logo = styled.img`
+  width: 210px;
```

Comment:
Can we use `rem` values for these?  Right now 1rem = 10px.

Reply:
okay